src/Halogen/Driver.purs

```diff
@@ -131,7 +131,7 @@ runUI c s element = _.driver <$> do
         when (isJust rp) $ render ref
         putVar rpRef Nothing
         q
-      HaltHF -> empty
+      HaltHF msg -> throwError $ error msg
```

Comment:
For reference: https://github.com/slamdata/purescript-aff/blob/f9b890130eb12e02b65d5ff5c6895acf393b1a8f/src/Control/Monad/Aff.purs#L195-L196